guess.js

```diff
@@ -147,14 +156,22 @@ function guessCheck() {
         // span.append(text)
         // massageArea.append(text2, span)
         //// OR esily///
+        // [REVIEW] Usually I suggest creating elements instead of directly setting the innerHTML.
+        // .. this is because it's easier to manage the elements and their attributes later.
+        // [REVIEW] It would be nice if you can create a little function to update the message area text ..
+        // since you are repeating the same code in multiple places.
         massageArea.innerHTML = `you win the word is <span>${wordGuess}</span>`
+        // [REVIEW] You can set another variable called maxTries and use it here to avoid ..
+        // .. the need to update this value if you decide to change the number of tries.
         if (numberOfHint === 2) {
             massageArea.innerHTML = `<p>congrantz you didn't us hints</p>`
         }
 
         //////add disabled class on ALL try Divs 
         let allTries = document.querySelectorAll(".inputs > div")
         // OR
+        // [REVIEW] You should always be more specific when selecting elements, so I recommend using ..
+        // .. .try instead of .inputs > div.

```

Comment:
why .try this class for div ?!

Reply:
a one way  can i wirt more div but class it more specific i think !

Reply:
`.try` gets all divs with `try` class, which is what you're trying to access here, right?
So instead of `.inputs > div`, it's better to just be more specific and use the given class name, which is `.try`.
Basically what you did in line 175 right below.

Reply:
ok i am add a class .try to div with class try-{$i} cuz i can select all div